modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/internal/util/JakartaCompatibilityUtil.java

```diff
@@ -24,14 +24,16 @@ public static boolean isUseJakarta(Project project) {
 			return true;
 		}
 
+		String product = workspaceExtension.getProduct();
+
 		String targetPlatformVersion =
 			workspaceExtension.getTargetPlatformVersion();
 
 		if (StringUtil.isBlank(targetPlatformVersion)) {
 			return false;
 		}
 
-		return VersionUtil.isJakartaCompatibleVersion(targetPlatformVersion);
+		return VersionUtil.isJakartaCompatibleVersion(product, targetPlatformVersion);
```

Comment:
is it not better to inline this since the function call is simple and only used once? Going based off of what Bchan would prefer